### DISCUSSION TITLE
Horizon Chase 2 [0100001019F6E000] v1.6.6 - 60FPS

### DISCUSSION THREAD

**Discussion by antoniodesousa** [StevensND/switch-port-mods#22] [Mod Request]
Could you update your 60FPS patch to v1.6.6? Thanks.

**Comment by StevensND** [comment]
Sure.  I will try to do it this week.

**Comment by StevensND** [comment]
> Could you update your 60FPS patch to v1.6.6? Thanks.

**[Done](https://github.com/StevensND/ue4-emuswitch-60fps/tree/main/Horizon%20Chase%202/%5B0100001019F6E000%5D)**

**Comment by antoniodesousa** [comment]
Nice, thanks! ❤️
